src/js/components/Menu/__tests__/__snapshots__/Menu-test.js.snap

```diff
@@ -437,6 +437,9 @@ exports[`Menu close by clicking outside 2`] = `
   border: none;
   padding: 0;
   text-align: inherit;
+  background-color: rgba(221,221,221,0.4);
+  color: #444444;
+  color: #000000;
```

Comment:
Please double-check why there are two different assignments of color CSS on the DOM.
lines #441, #442.
```
color: #444444;
color: #000000;
```

And why do color and background-color were impacted by this change?

Reply:
Color and background-color were impacted because previously, the menu items did not receive any changes to their styling to indicate that the menu item was in focus (aside from the focus outline). However, the behavior that Eric and I decided would be most appropriate according to online resources is that the menu item should receive the same styling as it would on hover. The background-color, I believe, is coming from when the menu item is the active element and the background changes to indicate this.  The color #444444 is the text color when the menu item is not in focus while #000000 is the text color when it is in focus. I agree that it seems strange to see two color assignments. How do hover states typically get displayed in the test snapshot?

Reply:
hover styling should appear like this:
```
.c1:hover {
...
}
```

I suspect there is more happening behind the scenes rather than hover, that assigns `color` twice with different values.

Reply:
Gotcha. Let me do some investigation.

Reply:
After looking more into things, it does seem that the reassignment on lines #440-#442 is coming from when the menu item becomes active. These lines adjust the styling to be the same as when a menu item is hovered (notice the same assignment occurs on #464-#466). The reason it is not appearing as a :hover for lines #440-#442 is because it is not a change in styling due to hover but instead to the prop "active" being enabled. 

Each menu item is a button, and a similar behavior can be seen in snapshots for button. If you go to src/js/components/Button/__tests__/__snapshots__/Button-test.js.snap and go to line #256-#266, you can see that there was an existing background and color styling that was then reassignment in the following lines.

The changes to the Menu snapshot seem to be aligning with other examples of styling that occurs when an element becomes active.

Reply:
Thank you for looking into it and giving a clear explanation.